README.md

```diff
@@ -84,6 +84,10 @@ In rare cases, you may want to also copy each dependency into the build product
 
 If you’ve modified your [Cartfile][], or you want to update to the newest versions of each framework (subject to the requirements you’ve specified), simply run the `carthage update` command again.
 
+### Transitive dependencies
```

Comment:
Trying to think of a better title but I'm stuck on "dependencies' dependencies" right now... maybe "Framework's dependencies" since we are in the "Adding frameworks to an application" section?